webrtc-stats.html

```diff
@@ -2699,6 +2700,16 @@ <h2>
                   not represent incoming data.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>silentConcealedSamples</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  The total number of concealed samples inserted that are silent. Playing out
+                  silent samples result in no audio. This is a subset of <a>concealedSamples</a>.
```

Comment:
So if the samples are generated as comfort noise with a non-zero volume, they are not counted as silent?
Nit: result in no audio -> results in no audio. Or perhaps "results in silence", since that's what's being played out (but makes it sound like a tautology).

Reply:
Still waiting to see what Chrome's implementation does, but this makes sense for now and PR updated, since what you are about to distinguish is between "no noise" and "noise or someone talking", the latter may result in robot voices, etc.

Reply:
I checked with Chrome - we count comfort noise as silence, and the spec is now clear on that point.